File: tools/dbench/answers/python_haiku/s1/f1_users.py
with open("users.txt") as f:
    lines = f.readlines()
valid_users = []
for line in lines:
    parts = line.strip().split()
    if len(parts) == 3:
        try:
            age = int(parts[2])
            valid_users.append(age)
        except ValueError:
            pass
print(len(valid_users))
if valid_users:
    print(sum(valid_users) // len(valid_users))
else:
    print(0)

File: tools/dbench/answers/rust_sonnet/s1/f1_users.rs
use std::fs;

fn main() {
    let contents = fs::read_to_string("users.txt").unwrap_or_default();
    let mut count = 0usize;
    let mut age_sum = 0i64;
    for line in contents.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i64>() {
                count += 1;
                age_sum += age;
            }
        }
    }
    println!("{}", count);
    if count > 0 {
        println!("{}", age_sum / count as i64);
    } else {
        println!("0");
    }
}
